### PR TITLE
Instagram embed

### PR DESCRIPTION
**Pull request by diegoquinteiro**
This PR

* [x] Adds two new getters for multiple elements and for sibling elements
* [x] Adds proper serialization of string tags (not self-closing).
* [x] Updates WordPress example rules to include up to date IG embed matching
* [x] Guarantees no silently fail when selecting any sibling element. 
* [ ] Migrate all embed rules to use this sibling selector

### CODE REVIEW

File: tests/Facebook/InstantArticles/Transformer/CMS/WPTransformerTest.php [hunk 1]
```diff
@@ -75,4 +75,23 @@ public function testTransformerLikeWPContent()
         // there must be 3 warnings related to <img> inside <li> that is not supported by IA
         $this->assertEquals(3, count($transformer->getWarnings()));
     }
+
+    public function testTitleTransformedWithBold()
```
Comment (diegoquinteiro) [inline tests/Facebook/InstantArticles/Transformer/CMS/WPTransformerTest.php]: I've moved this one to the WP tests as it uses WP rules.

File: src/Facebook/InstantArticles/Transformer/Getters/GetterFactory.php [hunk 2]
```diff
@@ -15,9 +15,11 @@ class GetterFactory
     const TYPE_CHILDREN_GETTER = 'children';
     const TYPE_ELEMENT_GETTER = 'element';
     const TYPE_NEXTSIBLING_GETTER = 'sibling';
+    const TYPE_NEXTSIBLINGELEMENT_GETTER = 'sibling-element';
```
Comment (everton-rosario) [inline src/Facebook/InstantArticles/Transformer/Getters/GetterFactory.php]: Why not TYPE_NEXT_SIBLING_ELEMENT_GETTER?

File: src/Facebook/InstantArticles/Transformer/Getters/MultipleElementsGetter.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+use Symfony\Component\CssSelector\CssSelectorConverter;
+
+class MultipleElementsGetter extends AbstractGetter
+{
+    /**
+     * @var Getters
+     */
+    protected $children = [];
+
+    // TODO: make it static
```
Comment (everton-rosario) [inline src/Facebook/InstantArticles/Transformer/Getters/MultipleElementsGetter.php]: I guess this TODO won't happen.

**Comment by everton-rosario** [comment]
All good stuff here. This is a good solution for the complex problem of the sibling selectors.

Lets add more tests:
- Checking the number of warnings
- Checking if the siblings are still raising warnings
- Selecting distant siblings (not side by side)
- Selecting more than 2 siblings (3 or more)

**Comment by diegoquinteiro** [comment]
looks good to me =)

**Pull request merged**